test/iron-image.html

```diff
@@ -207,6 +207,33 @@
 
             image.src = randomImageUrl();
           });
+
+          // Tests for PolymerElements/iron-image#115
+          test('image is not loaded when prevent-load is set', function() {
+            var attemptedLoadSpy = sinon.spy(image, '_setLoading');
```

Comment:
So here, I would basically be ensuring that no events were fired. I'm not sure how I could check that easily.

I was previously using a spy to check that it hadn't been called but that didn't seem reliable.

I will try another attempt at that.